docs/content/dagster-plus/deployment/azure/acr-user-code.mdx

```diff
@@ -0,0 +1,135 @@
+# Deploying user code in Azure Container Registry (ACR) with Dagster+
+
+This quickstart guide will walk you through setting up a new repository for your Dagster code, setting up CI/CD with GitHub Actions backed by Azure Container Registry (ACR), and deploying your code to your Azure Kubernetes Service (AKS) cluster.
+
+This guide assumes you already have an AKS agent running. You can follow along [here](/dagster-plus/deployment/azure/aks-agent) if you still need to set up an AKS agent.
+
+## Prerequisites
+
+This guide will use a Github repository to store the Dagster code, and GitHub Actions to deploy the code to Azure Container Registry - but you should be able to adapt these steps to other version control systems and CI/CD tools.
+
+- The azure CLI installed on your machine. You can download it [here](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli).
+- A GitHub account, and the ability to run GitHub Actions workflows in a repository.
+
+## Step 1: Creating a repository for Dagster code.
+
+We'll create a new repository based on the [Dagster+ hybrid quickstart repository](https://github.com/dagster-io/dagster-cloud-hybrid-quickstart). We'll go through these steps using a brand new repository in GitHub, but you should be able to easily adapt these steps to an existing repository or other version control systems.
```

Comment:
```suggestion
We'll create a new repository based on the [Dagster+ hybrid quickstart repository](https://github.com/dagster-io/dagster-cloud-hybrid-quickstart). We'll go through these steps using a brand new repository in GitHub, but you should be able to easily adapt these steps to an existing repository or other version control systems.
```

Not quite, the GitHub workflows is entirely implemented using Dagster Cloud Github actions. For the user to be able to roll their own on other CI/CD solutions, they'd have to follow this part of the guide: https://docs.dagster.io/dagster-plus/getting-started#other-cicd-provider.

Reply:
yea that's a really good point.

Reply:
It might be worth adding another note here that if they're using Azure DevOps pipelines for CI / CD, then they follow the "Other CI / CD provider" path [here](https://docs.dagster.io/dagster-plus/getting-started#step-4-configure-cicd-for-your-project)

Reply:
will do.